sdk/textanalytics/ai-text-analytics/test/utils/stringUnitOfLengthHelpers.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import { Entity, StringIndexType, TextAnalyticsClient } from "../../src";
+
+/**
+ * calls the recognizePiiEntities on the input document and checks wether the
+ * offset and length of the first pii entity matches the expected ones.
+ * @param client - a text analytics client
+ * @param doc - an input document to be processed by the text analytics service
+ * @param stringUnitOfLength - the string unit of length used by the service to calculate
```

Comment:
rename the class and other places were `stringUnitOfLength ` is still present

Reply:
Fixed.